src/mpyl/cli/build.py

```diff
@@ -149,9 +149,16 @@ def invoke(self, ctx):
     required=False,
     help="Combine results with previous run(s)",
 )
+@click.option(
+    "--projects",
+    "--p",
+    type=str,
+    required=False,
+    help="Comma separated list of the projects to build",
+)
 @click.pass_obj
 def run(
-    obj: CliContext, ci, all_, tag, stage, sequential
+    obj: CliContext, ci, all_, tag, stage, sequential, projects
```

Comment:
By the way I would consider using the `multiple` option in click: https://click.palletsprojects.com/en/7.x/options/#multiple-options

So we don't need to parse the list ourselves

Reply:
I personally prefer `-p job,ocpp,kong-sync` from `-p job -p ocpp -p kong-sync`, and allowing both actually makes more of a mess of the parse logic I would say?

Reply:
I didn't mean both. So if you prefer the single one it's also fine

Reply:
does it work with `-p 'job, ocpp,kong-sync'` as well?